app/src/main/java/de/westnordost/streetcomplete/about/ChangelogFragment.kt

```diff
@@ -76,14 +76,15 @@ private fun readChangelog(resources: Resources) =
 
 private fun addedLinks(description: String): String {
     return description
-        .replace(Regex("([\\s,(])#([0-9]*)")) { matchResult ->
+        .replace(Regex("([\\s,(])#([0-9]+)")) { matchResult ->
             val s = matchResult.groupValues[1]
             val issue = matchResult.groupValues[2]
             "$s<a href=\"https://github.com/westnordost/StreetComplete/issues/$issue\">#$issue</a>"
         }
-        .replace(Regex("([\\s,(])@(\\w*)")) { matchResult ->
+        .replace(Regex("([\\s,(])@(\\w[\\-\\w]*)([\\s,.\\)']|$)")) { matchResult ->
```

Comment:
```suggestion
        .replace(Regex("([\\s,(])@([\\w-]+)")) { matchResult ->
```

I think this is enough. See

https://regexper.com/#%28%5B%5Cs%2C%28%5D%29%40%28%5B%5Cw-%5D%2B%29

Yes this is more than is allowed, but the changelog is written by me, so I won't include invalid github user names because those can't contribute

Reply:
I mean by that argument, this will work too (most stuff includes _ in \w, which isn't in the GitHub list according to that repo):
```suggestion
        .replace(Regex("([\\s(])@([a-zA-Z\d-]+)")) { matchResult ->
```

As I can't see any reason you'd want `,@peternewman` gramatically it should be e.g. `@westnordost, @peternewman`.

We could add the `{0,38}` bit in, or are you not bothered about making it strict at all? I wasn't particularly expecting invalid usernames, but just it e.g. matching on other things by mistake.

Having looked at the current changelog, there is just `@username.` or `@username</li>` currently.